namd/src/colvarproxy_namd.C

```diff
@@ -160,14 +162,23 @@ int colvarproxy_namd::update_target_temperature()
 }
 
 
+
+void colvarproxy_namd::init_atoms_map()
+{
+  size_t const n_all_atoms = Node::Object()->molecule->numAtoms;
+  atoms_map.resize(n_all_atoms);
```

Comment:
The `resize` is immediately followed by `assign`, so I think the `resize` can be removed?

Reply:
Done